napari/_qt/qt_main_window.py

```diff
@@ -529,6 +548,8 @@ def _add_file_menu(self):
         self.file_menu.addAction(save_all_layers)
         self.file_menu.addAction(screenshot)
         self.file_menu.addAction(screenshot_wv)
+        self.file_menu.addAction(clipboard)
+        self.file_menu.addAction(clipboard_wv)
```

Comment:
I'm more used to thinking of copy actions as being present in an edit menu, instead of a filemenu, but i get the screenshot in general is in the file menu. What does @goanpeca think here?

Reply:
Agreed but it may be something for a different PR, since we would need to move other issues and make sure they are disabled if canvas is empty

Reply:
Interesting, I hadn't realized that, but that's fine, can move later

Reply:
Will open an issue to track this.